### PR TITLE
[zh] Sync 2023-10-10-cri-o-community-package-infrastructure.md blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync 2023-10-10-cri-o-community-package-infrastructure.md blog into Chinese:
```
content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
```

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
+---
+layout: blog
+title: "CRI-O 正在移动至 pkgs.k8s.io"
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:3]:
```suggestion
title: "CRI-O 迁移至 pkgs.k8s.io"
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+<!--
+**Author:** Sascha Grunert
+-->
+
+**译者**：Wilson Wu (DaoCloud)
+
+<!--
+The Kubernetes community [recently announced](/blog/2023/08/31/legacy-package-repository-deprecation/) that their legacy package repositories are frozen, and now they moved to [introduced community-owned package repositories](/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the [OpenBuildService (OBS)](https://build.opensuse.org/project/subprojects/isv:kubernetes). CRI-O has a long history of utilizing [OBS for their package builds](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o), but all of the packaging efforts have been done manually so far.
+-->
+Kubernetes 社区[最近宣布](/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/)他们的旧包仓库已被冻结，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:25]:
```suggestion
Kubernetes 社区[最近宣布](/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/)旧的软件包仓库已被冻结，
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+现在他们将被转移到由 [OpenBuildService（OBS）](https://build.opensuse.org/project/subprojects/isv:kubernetes)提供支持的[社区自有的包仓库](/blog/2023/08/15/pkgs-k8s-io-introduction)中。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:26]:
```suggestion
现在这些软件包将被迁移到由 [OpenBuildService（OBS）](https://build.opensuse.org/project/subprojects/isv:kubernetes)
提供支持的[社区自治软件包仓库](/blog/2023/08/15/pkgs-k8s-io-introduction)中。
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+There will be a deprecation phase for the existing packages, which is currently being [discussed in the CRI-O community](https://github.com/cri-o/cri-o/discussions/7315). The new infrastructure will only support releases of CRI-O `>= v1.28.2` as well as release branches newer than `release-1.28`.
+-->
+现有软件包将进入一个弃用阶段，目前正在
+[CRI-O 社区中讨论](https://github.com/cri-o/cri-o/discussions/7315)。
+新的基础设施将仅支持 CRI-O `>= v1.28.2` 的版本以及比 `release-1.28` 新的版本分支。
+
+<!--
+## How to use the new packages
+-->
+## 如何使用新包  {#how-to-use-the-new-packages}
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:50]:
```suggestion
## 如何使用新软件包  {#how-to-use-the-new-packages}
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+<!--
+In the same way as the Kubernetes community, CRI-O provides `deb` and `rpm` packages as part of a dedicated subproject in OBS, called [`isv:kubernetes:addons:cri-o`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o). This project acts as an umbrella and provides `stable` (for CRI-O tags) as well as `prerelease` (for CRI-O `release-1.y` and `main` branches) package builds.
+-->
+与 Kubernetes 社区一样，CRI-O 提供 `deb` 和 `rpm` 软件包作为 OBS 中专用子项目的一部分，
+被称为 [`isv:kubernetes:addons:cri-o`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o)。
+该项目充当保护伞，提供 `稳定`（针对 CRI-O 标签）以及 `预发布`（针对 CRI-O `release-1.y` 和 `main` 分支）包构建。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:57]:
```suggestion
CRI-O 是伞形项目的母体，用于构建 `stable`（针对 CRI-O 标记）以及 `prerelease`（针对 CRI-O `release-1.y` 和 `main` 分支）软件包。
```
保护伞 总觉得是个负面词 -_!

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+**稳定版本：**
+
+<!--
+- [`isv:kubernetes:addons:cri-o:stable`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable): Stable Packages
+  - [`isv:kubernetes:addons:cri-o:stable:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.29): `v1.29.z` tags
+  - [`isv:kubernetes:addons:cri-o:stable:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.28): `v1.28.z` tags
+-->
+- [`isv:kubernetes:addons:cri-o:stable`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable)：稳定软件包
+  - [`isv:kubernetes:addons:cri-o:stable:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.29 )：`v1.29.z` 标签
+  - [`isv:kubernetes:addons:cri-o:stable:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.28 )：`v1.28.z` 标签
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:71]:
```suggestion
  - [`isv:kubernetes:addons:cri-o:stable:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.29 )：`v1.29.z` 标记
  - [`isv:kubernetes:addons:cri-o:stable:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:stable:v1.28 )：`v1.28.z` 标记
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+<!--
+The package build inputs will undergo daily reconciliation and will be supplied by CRI-O's static binary bundles. These bundles are built and signed for each commit in the CRI-O CI, and contain everything CRI-O requires to run on a certain architecture. The static builds are reproducible, powered by [nixpkgs](https://github.com/NixOS/nixpkgs) and available only for `x86_64`, `aarch64` and `ppc64le` architecture.
+-->
+包构建输入将进行日常调谐，并将由 CRI-O 的静态二进制包提供。
+这些捆绑包是针对 CRI-O CI 中的每次提交构建和签名的，
+并且包含 CRI-O 在特定架构上运行所需的所有内容。静态构建是可重复的，
+由 [nixpkgs](https://github.com/NixOS/nixpkgs) 提供支持，
+并且仅适用于 `x86_64`、`aarch64` 以及 `ppc64le` 架构。
+
+<!-- The CRI-O maintainers will be happy to listen to any feedback or suggestions on the new packaging efforts! Thank you for reading this blog post, feel free to reach out to the maintainers via the Kubernetes [Slack channel #crio](https://kubernetes.slack.com/messages/CAZH62UR1) or create an issue in the [packaging repository](https://github.com/cri-o/packaging/issues). -->
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:277]:
```suggestion
<!--
The CRI-O maintainers will be happy to listen to any feedback or suggestions on the new packaging efforts! Thank you for reading this blog post, feel free to reach out to the maintainers via the Kubernetes [Slack channel #crio](https://kubernetes.slack.com/messages/CAZH62UR1) or create an issue in the [packaging repository](https://github.com/cri-o/packaging/issues).
-->
```

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-43452--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2023/10/10/cri-o-community-package-infrastructure/) looks all right.

/approve

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+
+**译者**：Wilson Wu (DaoCloud)
+
+<!--
+The Kubernetes community [recently announced](/blog/2023/08/31/legacy-package-repository-deprecation/) that their legacy package repositories are frozen, and now they moved to [introduced community-owned package repositories](/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the [OpenBuildService (OBS)](https://build.opensuse.org/project/subprojects/isv:kubernetes). CRI-O has a long history of utilizing [OBS for their package builds](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o), but all of the packaging efforts have been done manually so far.
+-->
+Kubernetes 社区[最近宣布](/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/)旧的软件包仓库已被冻结，
+现在这些软件包将被迁移到由 [OpenBuildService（OBS）](https://build.opensuse.org/project/subprojects/isv:kubernetes)
+提供支持的[社区自治软件包仓库](/blog/2023/08/15/pkgs-k8s-io-introduction)中。
+CRI-O 在利用 [OBS 进行软件包构建](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o)方面有着悠久的历史，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:28]: 很久以来，CRI-O 一直在 ... 

“悠久的历史”，有点儿过度了。

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+## How to use the new packages
+-->
+## 如何使用新软件包  {#how-to-use-the-new-packages}
+
+<!--
+In the same way as the Kubernetes community, CRI-O provides `deb` and `rpm` packages as part of a dedicated subproject in OBS, called [`isv:kubernetes:addons:cri-o`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o). This project acts as an umbrella and provides `stable` (for CRI-O tags) as well as `prerelease` (for CRI-O `release-1.y` and `main` branches) package builds.
+-->
+与 Kubernetes 社区一样，CRI-O 提供 `deb` 和 `rpm` 软件包作为 OBS 中专用子项目的一部分，
+被称为 [`isv:kubernetes:addons:cri-o`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o)。
+CRI-O 是伞形项目的母体，用于构建 `stable`（针对 CRI-O 标记）以及 `prerelease`（针对 CRI-O `release-1.y` 和 `main` 分支）软件包。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:58]:
```suggestion
这个项目是一个集合，提供 `stable`（针对 CRI-O 标记）以及 `prerelease`（针对 CRI-O `release-1.y` 和 `main` 分支）版本的软件包。
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 10]
```diff
@@ -0,0 +1,283 @@
...
+  - [`isv:kubernetes:addons:cri-o:prerelease:v1.29`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.29)：
+  [`release-1.29`](https://github.com/cri-o/cri-o/commits/release-1.29) 分支
+  - [`isv:kubernetes:addons:cri-o:prerelease:v1.28`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.28)：
+  [`release-1.28`](https://github.com/cri-o/cri-o/commits/release-1.28) 分支
+
+<!--
+There are no stable releases available in the v1.29 repository yet, because v1.29.0 will be released in December. The CRI-O community will also **not** support release branches older than `release-1.28`, because there have been CI requirements merged into `main` which could be only backported to `release-1.28` with appropriate efforts.
+-->
+v1.29 仓库中尚无可用的稳定版本，因为 v1.29.0 将于 12 月发布。
+CRI-O 社区也将**不**支持早于 `release-1.28` 的版本分支，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:97]:
```suggestion
CRI-O 社区也**不**支持早于 `release-1.28` 的版本分支，
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 11]
```diff
@@ -0,0 +1,283 @@
...
+#### 启动 CRI-O  {#start-cri-o}
+
+```bash
+systemctl start crio.service
+```
+
+<!--
+The Project's `prerelease:/main` prefix at the CRI-O's package path, can be replaced with `stable:/v1.28`, `stable:/v1.29`, `prerelease:/v1.28` or `prerelease:/v1.29` if another stream package is used.
+-->
+如果使用的是另一个流包，CRI-O 包路径中项目的 `prerelease:/main`
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:243]:
```suggestion
如果使用的是另一个包序列，CRI-O 包路径中项目的 `prerelease:/main`
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 12]
```diff
@@ -0,0 +1,283 @@
...
+前缀可以替换为 `stable:/v1.28`、`stable:/v1.29`、`prerelease:/v1.28` 或 `prerelease :/v1.29`。
+
+<!--
+Bootstrapping [a cluster using `kubeadm`](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) can be done by running `kubeadm init` command, which automatically detects that CRI-O is running in the background. There are also `Vagrantfile` examples available for [Fedora 38](https://github.com/cri-o/packaging/blob/91df5f7/test/rpm/Vagrantfile) as well as [Ubuntu 22.04](https://github.com/cri-o/packaging/blob/91df5f7/test/deb/Vagrantfile) for testing the packages together with `kubeadm`.
+-->
+可以通过运行 `kubeadm init` 命令来引导[使用 `kubeadm` 的集群](/docs/setup/product-environment/tools/kubeadm/install-kubeadm/)，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:249]:
```suggestion
你可以使用 `kubeadm init` 命令来[引导集群](/docs/setup/product-environment/tools/kubeadm/install-kubeadm/)，
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 13]
```diff
@@ -0,0 +1,283 @@
...
+该命令会自动检测后台正在运行 CRI-O。还有适用于
+[Fedora 38](https://github.com/cri-o/packaging/blob/91df5f7/test/rpm/Vagrantfile)
+以及 [Ubuntu 22.04](https://github.com/cri-o/packaging/blob/91df5f7/test/deb/Vagrantfile) 用于与 `kubeadm` 一起的测试包。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:252]:
```suggestion
以及 [Ubuntu 22.04](https://github.com/cri-o/packaging/blob/91df5f7/test/deb/Vagrantfile)
的 `Vagrantfile` 示例，可在使用 `kubeadm` 的场景中测试下载的软件包。
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 14]
```diff
@@ -0,0 +1,283 @@
...
+<!--
+## How it works under the hood
+-->
+## 它是如何工作的  {#how-it-works-under-the-hood}
+
+<!--
+Everything related to these packages lives in the new [CRI-O packaging repository](https://github.com/cri-o/packaging). It contains a [daily reconciliation](https://github.com/cri-o/packaging/blob/91df5f7/.github/workflows/schedule.yml) GitHub action workflow, for all supported release branches as well as tags of CRI-O. A [test pipeline](https://github.com/cri-o/packaging/actions/workflows/obs.yml) in the OBS workflow ensures that the packages can be correctly installed and used before being published. All of the staging and publishing of the packages is done with the help of the [Kubernetes Release Toolbox (krel)](https://github.com/kubernetes/release/blob/1f85912/docs/krel/README.md), which is also used for the official Kubernetes `deb` and `rpm` packages.
+-->
+与这些包相关的所有内容都位于新的 [CRI-O 打包仓库](https://github.com/cri-o/packaging)中。
+它包含 GitHub 的[每日调谐](https://github.com/cri-o/packaging/blob/91df5f7/.github/workflows/schedule.yml)操作工作流程，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:263]:
```suggestion
它包含 [Daily Reconciliation](https://github.com/cri-o/packaging/blob/91df5f7/.github/workflows/schedule.yml) GitHub 工作流，
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 15]
```diff
@@ -0,0 +1,283 @@
...
+支持所有发布分支以及 CRI-O 标签。
+OBS 工作流程中的[测试管道](https://github.com/cri-o/packaging/actions/workflows/obs.yml)确保包在发布之前可以被正确安装和使用。
+所有包的暂存和发布都是在 [Kubernetes 发布工具箱（krel）](https://github.com/kubernetes/release/blob/1f85912/docs/krel/README.md)的帮助下完成的，
+它也被用于官方 Kubernetes `deb` 和 `rpm` 软件包。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:267]:
```suggestion
这一工具箱也被用于官方 Kubernetes `deb` 和 `rpm` 软件包。
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 16]
```diff
@@ -0,0 +1,283 @@
...
+
+<!--
+The package build inputs will undergo daily reconciliation and will be supplied by CRI-O's static binary bundles. These bundles are built and signed for each commit in the CRI-O CI, and contain everything CRI-O requires to run on a certain architecture. The static builds are reproducible, powered by [nixpkgs](https://github.com/NixOS/nixpkgs) and available only for `x86_64`, `aarch64` and `ppc64le` architecture.
+-->
+包构建输入将进行日常调谐，并将由 CRI-O 的静态二进制包提供。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:272]:
```suggestion
包构建的输入每天都会被动态调整，并使用 CRI-O 的静态二进制包。
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md [hunk 17]
```diff
@@ -0,0 +1,283 @@
...
+这些捆绑包是针对 CRI-O CI 中的每次提交构建和签名的，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:273]:
```suggestion
这些包是基于 CRI-O CI 中的每次提交来构建和签名的，
```

File: content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md (hunk 8 above)
Comment (wilsonwu) [reply content/zh-cn/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md]: Done

**Review by my-git9 — COMMENTED** [review]
/lgtm

**Pull request merged**